pkg/rbac/common.go

```diff
@@ -77,16 +76,6 @@ func BuildSubjectFromRTB(object metav1.Object) (rbacv1.Subject, error) {
 		kind = "Group"
 	}
 
-	if sa != "" {
-		parts := strings.SplitN(sa, ":", 2)
-		if len(parts) < 2 {
-			return rbacv1.Subject{}, errors.Errorf("service account %s of projectroletemplatebinding is invalid: %v", sa, object.GetName())
-		}
-		namespace = parts[0]
-		name = parts[1]
-		kind = "ServiceAccount"
-	}
-
```

Comment:
Similar comment as on the project/subject key removal - I think we still need this code to remain so that we can remove the existing bindings through the framework.